server/src/domain/person/person.service.ts

```diff
@@ -189,7 +198,8 @@ export class PersonService {
   }
 
   private async findOrFail(authUser: AuthUserDto, id: string) {
-    const person = await this.repository.getById(authUser.id, id);
+    await this.access.requirePermission(authUser, Permission.PERSON_READ, id);
```

Comment:
I would prefer _not_ requiring this check here and _only_ having access checks at the beginning of the service methods. We don't need to check for person.read and then later (here) for person.write.